rules/prefer-todo.js

```diff
@@ -34,12 +34,20 @@ function removeSecondArg({ arguments: [first, second] }, fixer) {
   return fixer.removeRange([first.range[1], second.range[1]]);
 }
 
+function isFirstArgSpread({ arguments: [firstArg] }) {
+  return firstArg && firstArg.type === 'SpreadElement';
+}
+
 function create(context) {
   return {
     CallExpression(node) {
       if (isTestCase(node)) {
         const combineFixers = composeFixers(node);
 
+        if (isFirstArgSpread(node)) {
```

Comment:
you also need to somehow not touch https://github.com/facebook/jest/blob/6081f7a4f02f28a5446857bb9e33a54f84a61333/packages/jest-each/src/bind.js#L219-L220

that doesn't use spread

Reply:
Yeah, probably. It might also be fine to use ignores in jest's code base. But being on the safe side seems better.

Not sure if it makes sense to only autofix if there's a string, but still warn?

Reply:
I think developers only left tests empty with strings to make them later. If there is other than strings then they probably going to make some utils. I think we should apply this rule for strings only. What do yo think?

Reply:
Yeah, let's start with that